### PR TITLE
Add Hotkey for Users Profile

### LINKED ISSUE #688: Go to profile keyboard shortcut
Thanks so much for this browser extension!

Do we have any keyboard shortcut to go to my profile page from dashboard?
Currently, if we try <kbd>g</kbd> <kbd>p</kbd> hotkey, it take us to Pull Request.

It would be nice if we have some keyboard shortcut to go to the profile page.

Regards!

### PR DESCRIPTION
**Pull request by CrowderSoup**
Add a hotkey (`g m`) for the users profile.

Fixes #688

### CODE REVIEW

**Comment by CrowderSoup** [comment]
I went with `g m` for the hotkey because it didn't seem to like two key combo's. `g pf` was treated the same as `g p` (which is already in use).

File: src/content.js [hunk 1]
```diff
@@ -148,6 +148,19 @@ async function addTrendingMenuItem() {
...
 
+async function addProfileHotkey() {
+	const userMenuExists = select.exists('#user-links');
+	if (!userMenuExists) {
+		return;
+	}
+
+	const menuItem = select.all('#user-links a.dropdown-item').filter(item => {
+		return item.text.trim().toLowerCase() === 'your profile';
+	});
```
Comment (bfred-it) [inline src/content.js]: You can select it directly with `'[href="/${getUsername()}"]'` instead of the previous 7 lines.

File: src/content.js [hunk 2]
```diff
@@ -148,6 +148,19 @@ async function addTrendingMenuItem() {
...
+
+	$(menuItem).attr('data-hotkey', 'g m');
```
Comment (bfred-it) [inline src/content.js]: -> `.setAttribute()`, no jQuery required

**Comment by bfred-it** [comment]
Thanks! The only thing is that the function doesn't need to be `async`

**Comment by CrowderSoup** [comment]
@bfred-it true. I'm used to writing all async functions at work so it's become a habit, haha!

**Comment by CrowderSoup** [comment]
@bfred-it just pushed that change! You should be good to merge any time.

**Comment by hkdobrev** [comment]
Cheers!

**Pull request merged**